templates/admin/pages/button_settings.php

```diff
@@ -96,13 +96,13 @@ class="wcp-rquery"
         <div  id='wc_call_for_price__upload_image_wrapper'>
             <?php if( ! empty( $upload_image_url ) ) : ?>
             <div class="wcp-uploaded-image-preview_wrapper">
-                <img src="<?php echo esc_attr( $upload_image_url );?>" alt="">
+                <img src="<?php echo esc_attr( $upload_image_url );?>" alt="Button Image" id="wc_call_for_price_image_preview">
```

Comment:
`alt="<?php echo esc_attr( 'Button Image', 'wc-call-for-price' ); ?>"`
make `alt` attribute translatable.